### PR TITLE
[settings] Use pi-ui's radio buttons in timezone settings

### LINKED ISSUE #2653: [pi-ui][settings] Use pi-ui's radio buttons instead of custum ones.
image

### PR DESCRIPTION
**Pull request by amassarwi**
This diff replaces custom radio buttons used in timezone settings with pi-ui's component.
Closes #2653 
Needs https://github.com/decred/pi-ui/pull/293

---

Before:
image

After:
image

### CODE REVIEW

**Comment by amassarwi** [comment]
I decided to expand pi-ui's component and add the description field to perfectly match the existing specs.

See https://github.com/decred/pi-ui/issues/290

File: package.json [hunk 1]
```diff
@@ -283,7 +283,7 @@
     "node-abi": "^2.15.0",
     "node-addon-loader": "decred/node-addon-loader#master",
     "nouislider": "^12.0.0",
-    "pi-ui": "https://github.com/decred/pi-ui",
+    "pi-ui": "https://github.com/amassarwi/pi-ui#rbDescription",
```
Comment (amassarwi) [inline package.json:286]: dev/test purposes, will revert and upgrade pi-ui when https://github.com/decred/pi-ui/pull/293 is merged.

Comment (vctt94) [reply package.json:286]: Can you revert this, so we can merge it?

**Review by vctt94 — APPROVED** [review]
LGTM. After removing the change on pi-ui's directory is good to go!

**Comment by amassarwi** [comment]
@vctt94 done mate

**Review by alexlyp — APPROVED** [review]
ACK

**Pull request merged**